lib/ssoe/configuration.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/base'
+
+module SSOe
+  class Configuration < Common::Client::Configuration::SOAP
+    def service_name
+      'SSOe'
+    end
+
+    def connection
+      @connection ||= Faraday.new(base_path,
+                                  headers: base_request_headers,
+                                  request: request_options,
+                                  ssl: ssl_options) do |faraday|
+        faraday.request :soap_headers
```

Comment:
Can you add a ticket to add 'breakers'?  We don't need it in this PR but we'll likely want to eventually have it

Breakers makes it so when the system is down, we don't keep hammering it, basically a circuit breaker